nss-day-cohort-51/nss-day-cohort-51.github.io
# Ss cleanup 139

# Description - General code clean up and resizing styling

## Type of change

- [x] Cleaned up unnecessary code and comments
- [x] updated description section media queries and styling

# Testing Instructions
Fetch branch and serve. Resize page and watch description section to make sure demo day button doesn't touch bg image swoop and the words Full Stack, Tech Stack, Cohort 51, and February 11th, 2022 don't split when resizing

# Checklist:
